src/System.Management.Automation/DscSupport/CimDSCParser.cs

```diff
@@ -1666,12 +2127,16 @@ private static ParseError[] CheckMandatoryPropertiesPresent(DynamicKeywordStatem
                             }
                             catch (FileNotFoundException)
                             {
-                                errorList.Add(new ParseError(scriptExtent, "SchemaFileNotFound", string.Format(CultureInfo.CurrentCulture, ParserStrings.SchemaFileNotFound, schemaScriptFilePath)));
+                                errorList.Add(new ParseError(scriptExtent,
+                                                             "SchemaFileNotFound",
+                                                             string.Format(CultureInfo.CurrentCulture, ParserStrings.SchemaFileNotFound, schemaScriptFilePath)));
                             }
                             catch (Exception e)
                             {
                                 // This shouldn't happen so just report the error as is
-                                errorList.Add(new ParseError(scriptExtent, "UnexpectedParseError", string.Format(CultureInfo.CurrentCulture, e.ToString())));
+                                errorList.Add(new ParseError(scriptExtent,
+                                                             "UnexpectedParseError",
+                                                             string.Format(CultureInfo.CurrentCulture, e.ToString())));
```

Comment:
Same as above

Reply:
Also `e.ToString()` shouldn't be necessary -- that's called by `string.Format` on object arguments